lib/core/src/lib/form/components/widgets/date-time/date-time.widget.ts

```diff
@@ -42,68 +43,115 @@ import { WidgetComponent } from '../widget.component';
     ],
     templateUrl: './date-time.widget.html',
     styleUrls: ['./date-time.widget.scss'],
-    imports: [NgIf, TranslateModule, MatFormFieldModule, MatInputModule, MatDatetimepickerModule, FormsModule, ErrorWidgetComponent],
+    imports: [NgIf, TranslateModule, MatFormFieldModule, MatInputModule, MatDatetimepickerModule, ReactiveFormsModule, ErrorWidgetComponent],
```

Comment:
please make it onPush

Reply:
Agree, but it would be better to do it in separate task, issue -> https://hyland.atlassian.net/browse/AAE-24759